examples/bri-3/src/bri/merkleTree/agents/merkleTree.agent.ts

```diff
@@ -15,8 +15,8 @@ export class MerkleTreeAgent {
   public createNewMerkleTree(leaves: string[]): BpiMerkleTree {
     return new BpiMerkleTree(
       v4(),
-      'sha256',
-      this.service.formMerkleTree(leaves, 'sha256'),
+      `${process.env.HASH_ALGORITHM}`,
```

Comment:
could you pass process.env.HASH_ALGORITHM directly, without quotes and ${} ?

Reply:
![image](https://github.com/eea-oasis/baseline/assets/57068961/bec8910b-d585-4668-8eb0-b44b95cbf959)


As you can see, I have to convert process.env var to string because the class should receive the string parameter.
I think `${}` is the most readable way, but I also can use `process.env.VAR_NAME as string` or extract process.env var to global var and later use it.
Would you happen to have some other solution or suggestion?

Reply:
Ok, that is because of the strict setting complaining that the env could be undefined. Could be solved by adding ! at the end of the variable to tell the linter that the value will never be undefined, but your approach works as well. Approving.